cmd/livepeer_cli/wizard_stats.go

```diff
@@ -194,11 +194,10 @@ func (w *wizard) orchestratorStats() {
 		[]string{"Delegated Stake", eth.FormatUnits(t.DelegatedStake, "LPT")},
 		[]string{"Reward Cut (%)", eth.FormatPerc(t.RewardCut)},
 		[]string{"Fee Share (%)", eth.FormatPerc(t.FeeShare)},
-		[]string{"Price Per Segment", eth.FormatUnits(t.PricePerSegment, "ETH")},
 		[]string{"Pending Reward Cut (%)", eth.FormatPerc(t.PendingRewardCut)},
 		[]string{"Pending Fee Share (%)", eth.FormatPerc(t.PendingFeeShare)},
-		[]string{"Pending Price Per Segment", eth.FormatUnits(t.PendingPricePerSegment, "ETH")},
 		[]string{"Last Reward Round", t.LastRewardRound.String()},
+		[]string{"Base price per pixel", priceInfo.Num().String(), "wei /", priceInfo.Denom().String(), "pixels"},
```

Comment:
This ends up rendering like this:

<img width="566" alt="Screen Shot 2019-07-22 at 3 46 35 PM" src="https://user-images.githubusercontent.com/5933273/61660487-ed225480-ac97-11e9-9049-a9ed5096aaf8.png">

We really want a single concatenated string table entry instead of multiple strings with multiple table entries. Something like this should suffice:

```
fmt.Sprintf("%v wei / %v pixels", priceInfo.Num(), priceInfo.Denom())
```

Reply:
fixed in 894375e